tests/unit/worker/test_submission_worker.py

```diff
@@ -1,28 +1,77 @@
 import boto3
+import mock
 import os
 import shutil
 import tempfile
 
+from datetime import timedelta
 from moto import mock_sqs
 from os.path import join
-from unittest import TestCase
 
+from django.contrib.auth.models import User
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.utils import timezone
+
+from rest_framework.test import APITestCase
+
+from challenges.models import (
+    Challenge,
+    ChallengePhase,
+)
+from hosts.models import ChallengeHostTeam
+from jobs.models import Submission
+from participants.models import ParticipantTeam
 from scripts.workers.submission_worker import (
     create_dir,
     create_dir_as_python_package,
+    extract_submission_data,
     return_file_url_per_environment,
     get_or_create_sqs_queue
 )
 
 
-class BaseAPITestClass(TestCase):
+class BaseAPITestClass(APITestCase):
     def setUp(self):
+
         self.BASE_TEMP_DIR = tempfile.mkdtemp()
+        self.COMPUTE_DIRECTORY_PATH = join(
+            self.BASE_TEMP_DIR, "compute"
+        )
+        self.CHALLENGE_DATA_BASE_DIR = join(
+            self.COMPUTE_DIRECTORY_PATH, "challenge_data"
+        )
+        self.SUBMISSION_DATA_BASE_DIR = join(
```

Comment:
@nikochiko I think creating just 1 variable for `/compute/challenge_data/submission_files` prefix is enough. I don't think we will need separate variables for `CHALLENGE_DATA_BASE_DIR` or `COMPUTE_DIRECTORY_PATH` for now.